cli/compose/convert/service.go

```diff
@@ -217,13 +217,17 @@ func convertServiceSecrets(
 		if gid == "" {
 			gid = "0"
 		}
+		mode := secret.Mode
+		if mode == 0 {
```

Comment:
Is this the right check? Mode 0 should be settable since it exists as a mode. If I tried to set the mode to 0 I'd be surprised if it ended up 444.